lib/homework_1/task_1.js

```diff
@@ -0,0 +1,10 @@
+import readline from "readline";
```

Comment:
You script is duplicating input to console. Here is output:
"test
test
tset
12345  54321  
12345  54321  
  12345  54321
"
Please do some corrections to see input once as in task description. Example:
"12345
54321"

Reply:
Fixed. Thank you.